pkgs/desktops/deepin/artwork/dde-account-faces/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "dde-account-faces";
+  version = "1.0.12.1";
+
+  src = fetchFromGitHub {
+    owner = "linuxdeepin";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-NWjR8qxWi2IrcP0cSF+lLxBJ/GrVpk1BfTjVH0ytinY=";
+  };
+
+  # It should be installed to /var，but this can't be done directly on nixos, so move to $out/share
+  # we need patch dde-control-center / dde-daemon also
+  makeFlags = [
+    "DESTDIR=$(out)/share"
+    "PREFIX=/"
+  ];
```

Comment:
Double checked this part. So we are installing the icons into `$out/share/lib/AccountsService`, How does other packages (like the gnome ones) handle this?

Reply:
pinging @jtojnar in case you happen to know.

Reply:
Do not use `DESTDIR`: https://github.com/NixOS/nixpkgs/issues/65718